standard.mk

```diff
@@ -29,7 +29,7 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
-TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
+TESTTARGETS := ./pkg/controller/secret
```

Comment:
Why hardcode it to this? the `go list ...` stuff will find all of the packages, and if they have no tests `make gotest` will skip, and that' ok.

Reply:
I was getting docker build failures during the test phase, so I thought I needed to change this. But I just tested it out with the old code and it works now. I must have been hitting a different problem.